src/Components/MovieList/MovieSearch.test.js

```diff
@@ -27,24 +27,13 @@ test("api call success", async () => {
     poster: "somePoster.jpg",
   };
 
-  const fakeFunction = () => {
-    return new Promise((resolve, reject) => {
-      resolve([movie1]);
-    });
-  };
-
-  const { getByTestId, getByText } = render(
-    act(() => {
-      <MovieSearch fetchMovies={fakeFunction} />;
-    })
-  );
-  // fireEvent(getByTestId("movie-fetch"), new MouseEvent("click"));
+  const fakeFunction =() => Promise.resolve([movie1]);
 
-  act(() => {
-    fireEvent.click(getByText("Search"));
-  });
+  const { getByTestId, getByText } = render(<MovieSearch fetchMovies={fakeFunction} />);
+  fireEvent.click(getByText("Search"));
 
-  // 3
+  await waitForElementToBeRemoved(() => screen.getByTestId("no-movie-response"))
```

Comment:
This is a react testing lib helper, does what it says, waits till the `no-movie-response` item is not in the DOM anymore before moving on to our assertion below